content_service.go

```diff
@@ -49,7 +49,7 @@ type Entry struct {
 	Path       string       `json:"path"`
 	Type       EntryType    `json:"type"` // can be JSON, TEXT or DIRECTORY
 	Content    EntryContent `json:"content,omitempty"`
-	Revision   string       `json:"revision,omitempty"`
+	Revision   int64        `json:"revision,omitempty"`
```

Comment:
Actually, we use signed int32 in upstream. How about making this as same as upstream?

Reply:
Ah I saw your commit message after I left this comment.